### PR TITLE
[Hack Update] 047-TrafficControlWithDapr - Dapr conduent updates

### PR DESCRIPTION
**Pull request by jordanbean-msft**

### CODE REVIEW

File: 047-TrafficControlWithDapr/Coach/Solution-07.md [hunk 1]
```diff
@@ -47,23 +45,23 @@ First, you'll add a secrets management JSON configuration file to the solution:
...
+    _Understand that if you specify a relative path to the `secretsFile` (as is the case here), this path must be specified relative to the folder where you start your service using the Dapr CLI. Because you start the services from their project folders, the relative path to the components folder is always `../dapr/components`._
 
-The `nestedSeparator` in the `metadata` section specifies the character that Dapr will use when it flattens the secret file's hierarchy. Each secret will be uniquely identifiable by one key. In this case, you're using the period (`.`) as that character. The convention means that secrets from the `secrets.json` file will be identified by the following keys:
+The `nestedSeparator` in the `metadata` section specifies the character that Dapr will use when it flattens the secret file's hierarchy. Each secret will be uniquely identifiable by one key. In this case, you're using the period (`-`) as that character. The convention means that secrets from the `secrets.json` file will be identified by the following keys:
```
Comment (perktime) [inline 047-TrafficControlWithDapr/Coach/Solution-07.md:60]: Do you mean "you're using the hyphen as that character"?

**Review by perktime — CHANGES_REQUESTED** [review]

File: 047-TrafficControlWithDapr/Coach/Solution-07.md (hunk 1 above)
Comment (jordanbean-msft) [reply 047-TrafficControlWithDapr/Coach/Solution-07.md:60]: yes, that should be changed

File: 047-TrafficControlWithDapr/Student/Challenge-00.md [hunk 2]
```diff
@@ -111,7 +215,7 @@ You'll create the Azure resources for the subsequent challenges using [Azure Bic
     iotHubName:
       type: String
       value: iothub-dapr-mce123
-    keyVaultName:
+    Key VaultName:
```
[the remark this answers is not in the document]
Comment (jordanbean-msft) [reply 047-TrafficControlWithDapr/Student/Challenge-00.md:218]: no, looks like a find/replace error, fixing...

File: 047-TrafficControlWithDapr/Coach/Solution-07.md (hunk 1 above)
Comment (jordanbean-msft) [reply 047-TrafficControlWithDapr/Coach/Solution-07.md]: fixed

**Review by perktime — APPROVED** [review]

**Review by perktime — APPROVED** [review]

File: 047-TrafficControlWithDapr/Student/Challenge-04.md [hunk 3]
```diff
@@ -60,5 +61,5 @@ This challenge targets the operation labeled as **number 3** in the end-state se
 - [Introduction to the state management building block](https://docs.dapr.io/developing-applications/building-blocks/state-management/)
 - [Dapr Redis State Store](https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-redis/)
 - [Redis CLI commands](https://redis.io/topics/rediscli)
-- [Configure Azure Redis for cach in Dapr](https://docs.dapr.io/getting-started/configure-state-pubsub/#tabs-3-azure)
+- [Configure Azure Cache for Redis in Dapr](https://docs.dapr.io/getting-started/configure-state-pubsub/#tabs-3-azure)
```
Comment (Whowong) [inline 047-TrafficControlWithDapr/Student/Challenge-04.md:64]: This link doesn't work, can you find a new one to replace it?

Comment (jordanbean-msft) [reply 047-TrafficControlWithDapr/Student/Challenge-04.md]: fixed

**Pull request merged**